hazelcast/src/test/java/com/hazelcast/scheduledexecutor/ScheduledExecutorServiceBasicTest.java

```diff
@@ -250,59 +236,52 @@ public void handlerTaskAndSchedulerNames_withRunnable()
     }
 
     @Test
-    public void stats()
-            throws ExecutionException, InterruptedException {
+    public void stats() throws Exception {
         double delay = 2.0;
 
         HazelcastInstance[] instances = createClusterWithCount(2);
         Object key = generateKeyOwnedBy(instances[1]);
         IScheduledExecutorService executorService = getScheduledExecutor(instances, "s");
-        IScheduledFuture<Double> future = executorService.scheduleOnKeyOwner(
-                new PlainCallableTask(), key, (int) delay, SECONDS);
+        IScheduledFuture<Double> future = executorService.scheduleOnKeyOwner(new PlainCallableTask(), key, (int) delay, SECONDS);
 
         future.get();
         ScheduledTaskStatistics stats = future.getStats();
 
         assertEquals(1, stats.getTotalRuns());
-        assertNotNull(stats.getLastIdleTime(SECONDS));
```

Comment:
Actually these asserts could never fail, since the stats return primitive longs, which are never `null`. So I replaced those with actual number asserts (I also tested them with `NANOSECONDS`, to be sure there is no timing dependency introduced).